core/cloudflow-akka/src/main/resources/reference.conf

```diff
@@ -10,3 +10,11 @@ akka.kafka.consumer.stop-timeout = 0s
 # Configure the time to wait after stopping the inflow before shutting down the Kafka consumer
 # This helps to process in-flight messages (and commits).
 cloudflow.akka.consumer-stop-timeout = 10 s
+
+# logback configuration
+# https://doc.akka.io/docs/akka/current/logging.html#slf4j
+akka {
```

Comment:
FYI Patrik has pointed out that overrides of reference.confs in a reference.conf does not always work. (it works so far in Cloudflow but that is likely because of ordering)